src/client/components/molecules/profileSteps/contact.tsx

```diff
@@ -5,73 +5,70 @@ type contactProps = {
   date: Portfolio;
 };
 
-const Resume = ({ date }: contactProps) => {
-  console.log();
-  return (
-    <div className="contact">
-      <div className="dvContainer">
-        <div className="dvContainer__title">OverView Date</div>
-        <hr className="dvContainer__line" />
-        <div className="dvContainer__container">
-          <div className="dvContainer__subtitle">
-            User Data
-            <hr className="dvContainer__overline" />
+const Resume = ({ date }: contactProps) => (

```

Comment:
again, probably was added in the past but. Can you please update the propType name to be like:
<ComponentName>Props
suggestion: `ResumeProps`